skt/executable.py

```diff
@@ -443,11 +445,19 @@ def cmd_run(cfg):
     run_report_path = join_with_slash(cfg.get('output_dir'), 'run.report')
 
     runner = skt.runner.getrunner(*cfg.get('runner'))
-    retcode, report_string = runner.run(cfg.get('buildurl'),
-                                        cfg.get('max_aborted_count'),
-                                        cfg.get('krelease'),
-                                        cfg.get('wait'),
-                                        arch=cfg.get("kernel_arch"))
+
+    atexit.register(runner.cleanup_handler)
+    signal.signal(signal.SIGINT, runner.signal_handler)
+    try:
+        retcode, report_string = runner.run(cfg.get('buildurl'),
+                                            cfg.get('max_aborted_count'),
+                                            cfg.get('krelease'),
+                                            cfg.get('wait'),
+                                            arch=cfg.get("kernel_arch"))
+    except Exception:
+        exc_info = sys.exc_info()
+        print('\n'.join(traceback.format_tb(exc_info[2])))
```

Comment:
Please use the initialized `logging` module instead of prints. Since this part of code is hiding infrastructure issues, please also either raise the caught exception or set the `retcode` to `SKT_ERROR`.